iamlistening/handler/client.py

```diff
@@ -100,7 +100,7 @@ async def handle_message(self, message_content):
 
         if self.is_connected:
             self.latest_message = message_content
-            logger.debug("Frasier👂 on {}: {}", self.platform, message_content)
```

Comment:
<a href="https://sourcery.ai"><img src="https://sourcery.ai/favicon.svg" height="15" alt="Sourcery logo" /></a>  **suggestion (code_refinement):** Consider implementing a more flexible logging level control.

Commenting out the debug log line reduces the ability to debug issues. It might be more beneficial to control the verbosity through logging configuration.